app_template_dev.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+"""Template for CDK Application"""
+# Standard
+import os
+
+# Installed
+from aws_cdk import App, Environment
+
+# Local
+from sds_data_manager.utils.stackbuilder import build_sds
+
+"""
+    This app is designed for individual developer testing.
```

Comment:
```suggestion
    This app is used for individual developer testing, instead of app.py. When deploying locally, you can update the sds_id to include your initials and distinguish your stack from the main dev stack or other developers.
```